### PR TITLE
Added "debug:block" command alias

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added `debug:sonata:block` command alias for `DebugBlocksCommand`
```

## Subject

You should use the `debug:` prefix for debug commands.

### CODE REVIEW

File: src/Command/DebugBlocksCommand.php [hunk 1]
```diff
@@ -23,6 +23,8 @@ class DebugBlocksCommand extends BaseCommand
      */
     public function configure()
     {
+        // NEXT_MAJOR: Switch name and alias
+        $this->setAliases(['debug:block']);
```
Comment (greg0ire) [inline src/Command/DebugBlocksCommand.php]: shouldn't it be `debug:sonata-block`?

Comment (OskarStark) [reply src/Command/DebugBlocksCommand.php]: Or debug:sonata:block?

Comment (greg0ire) [reply src/Command/DebugBlocksCommand.php]: Yeah, either one

**Comment by core23** [comment]
Changed this to `debug:sonata:block`

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**